cypress/fixtures/constants.ts

```diff
@@ -457,7 +463,7 @@ export class Constants {
         nextjsSsrDocumentationUrl: 'https://nextjs.org/docs',
         nextjsSsrLearnUrl: 'https://nextjs.org/learn/foundations/about-nextjs',
         nextjsSsrExamplesUrl: 'https://github.com/vercel/next.js/tree/deprecated-main/examples',
-        nativeFederationReactUrl: 'https://reactjs.org',
```

Comment:
please be careful with deleting. I used this link in one of my samples. revert deletion

Reply:
thank you. reverted